openedx_tagging/core/tagging/rest_api/v1/serializers.py

```diff
@@ -0,0 +1,19 @@
+"""
+API Serializers for taxonomies
+"""
+
+from rest_framework import serializers
+
+
+class TaxonomySerializer(serializers.Serializer):
```

Comment:
Using [ModelSerializer](https://www.django-rest-framework.org/api-guide/serializers/#modelserializer)s will reduce the lines of code required here.